### PR TITLE
Add HTTPs support to cluster.sh, healthcheck.sh and cp-subsystem.sh

### PR DESCRIPTION
**Pull request by kwart**
This PR adds HTTPs support to `cluster.sh`, `healthcheck.sh` and `cp-subsystem.sh` management scripts.

Several new arguments are introduced. Their naming mostly copies names in `curl`.

New arguments:

| Argument | Default value | Description |
| ------------- | ------------- | -----------------|
| `--https` | *no argument expected* | Uses HTTPs protocol for REST calls.
| `--cacert` | *set of well-known CA certificates* | Defines trusted PEM-encoded certificate file path. It’s used to verify member certificates.
| `--cert` | None | Defines PEM-encoded client certificate file path. Only needed when client certificate authentication is used.
| `--key` | None | Defines PEM-encoded client private key file path. Only needed when client certificate authentication is used.
| `--insecure` | *no argument expected* | Disables member certificate verification.
| `-d`, `--debug` | *no argument expected* | Prints `curl` error output.

### CODE REVIEW

**Comment by kwart** [comment]
run-lab-run

File: hazelcast/src/main/resources/cluster.sh [hunk 1]
```diff
@@ -71,50 +100,41 @@ fi
...
 fi
 
-MSG_FORBIDDEN="Please make sure you provide valid group name (or valid username/password combination when the Hazelcast security is enabled)."
-
 if [ "$OPERATION" = "get-state" ]; then
-    echo "Getting cluster state on ip ${ADDRESS} on port ${PORT}"
-	request="http://${ADDRESS}:${PORT}/hazelcast/rest/management/cluster/state"
- 	response=$(curl --data "${GROUPNAME}&${PASSWORD}" --silent "${request}");
+    echo "Getting cluster state on address ${ADDRESS}:${PORT}"
+    response=$(${CURL_CMD} --data "${GROUPNAME}&${PASSWORD}" "${URL_BASE}/state");
```
Comment (vojtechtoman) [inline hazelcast/src/main/resources/cluster.sh]: minor: is the trailing semicolon necessary here? Occurs in more places, and I see it's been in the older version, too.

File: hazelcast/src/main/resources/cp-subsystem.sh [hunk 2]
```diff
@@ -1,42 +1,49 @@
...
+    echo "  -c, --group       : Name of the CP group. Must be provided for 'get-group', 'force-destroy-group', 'get-sessions', 'force-close-session'."
+    echo "  -m, --member      : UUID of the CP member. Must be provided for 'remove-member'."
+    echo "  -s, --session-id  : CP Session ID. Must be provided for 'force-close-session'."
+    echo "  -a, --address     : Defines which ip address hazelcast is running. Default value is '127.0.0.1'."
+    echo "  -p, --port        : Defines which port hazelcast is running. Default value is '5701'."
+    echo "  -g, --groupname   : Defines groupname of the cluster. Default value is 'dev'."
+    echo "  -P, --password    : Defines password of the cluster. Default value is 'dev-pass'."
+    echo "  -d, --debug       : Prints curl error output."
+    echo "HTTPs related (TLS enabled):"
+    echo "      --https       : Uses HTTPs protocol for REST calls.(no parameter value expected)"
```
Comment (vojtechtoman) [inline hazelcast/src/main/resources/cp-subsystem.sh]: missing space before opening parenthesis

File: hazelcast/src/main/resources/healthcheck.sh [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
     echo "Parameters: "
-    echo "	-o, --operation	    : Health check operation. Operation can be 'all', 'node-state','cluster-state','cluster-safe','migration-queue-size','cluster-size'."
-    echo "	-a, --address  	    : Defines which ip address hazelcast node is running on. Default value is '127.0.0.1'."
-    echo "	-p, --port  	    : Defines which port hazelcast node is running on. Default value is '5701'."
+    echo "  -o, --operation   : Health check operation. Operation can be 'all', 'node-state','cluster-state','cluster-safe','migration-queue-size','cluster-size'."
+    echo "  -a, --address     : Defines which ip address hazelcast node is running on. Default value is '127.0.0.1'."
+    echo "  -p, --port        : Defines which port hazelcast node is running on. Default value is '5701'."
+    echo "  -d, --debug       : Prints curl error output."
+    echo "HTTPs related (TLS enabled):"
+    echo "      --https       : Uses HTTPs protocol for REST calls.(no parameter value expected)"
```
Comment (vojtechtoman) [inline hazelcast/src/main/resources/healthcheck.sh]: missing space before opening parenthesis

**Comment by kwart** [comment]
Thanks for reviews, guys. I'll address the comments.

**Pull request merged**